components/Storyblok/SbHomepageMvp.tsx

```diff
@@ -1,10 +1,10 @@
-import { storyblokEditable, type SbBlokData } from '@storyblok/react/rsc';
-import { CreateBloks } from '../CreateBloks';
-import { Heading } from '../Typography';
-import { HomepageSplitHero } from '../Homepage/HomepageSplitHero';
-import { TogetherSection } from '../Homepage/TogetherSection';
-import { Masthead } from '../Masthead';
-import { Changemaker } from '../Homepage/Changemaker';
+import { storyblokEditable, type SbBlokData, type ISbStoryData } from '@storyblok/react/rsc';
+import { CreateBloks } from '@/components/CreateBloks';
```

Comment:
Thanks for changing everything to the `@/components` convention. I like that much more.

Reply:
Yeah I edit them whenever I have to update a file with the old convention. I like the new one more because it lets one copy and paste between files in nested directory easily 😄